make/mt-os/mt-linux.mk

```diff
@@ -13,15 +13,12 @@ include make/mt-os/mt-os-common.mk
 
 ifdef XI_SHARED
   AR = gcc
-  XI ?= $(XI_BINDIR)/libxively.so
+  XI = $(XI_BINDIR)/libxively.so
```

Comment:
I've made it default to:
XI ?= $(XI_BINDIR)/libxively.a
in mt-os-common.mk. So in case of shared lib we should override.
Having default value avoid the need to set it during a porting process. What do you think is shared lib a practical form, is it needed, who uses this form?